contracts/contracts/token/WOETH.sol

```diff
@@ -54,18 +59,16 @@ contract WOETH is ERC4626, Governable, Initializable {
      *         as a governance operation.
      */
     function initialize2() public onlyGovernor {
-        require(!_oethCreditsInitialized, "Initialize2 already called");
-
-        _oethCreditsInitialized = true;
-        /*
-         * This contract is using creditsBalanceOfHighres rather than creditsBalanceOf since this
-         * ensures better accuracy when rounding. Also creditsBalanceOf can be a little
-         * finicky since it reports Highres version of credits and creditsPerToken
-         * when the account is a fresh one. That doesn't have an effect on mainnet since
-         * WOETH has already seen transactions. But it is rather annoying in unit test
-         * environment.
-         */
-        oethCreditsHighres = _getOETHCredits();
+        require(_adjuster == 0, "Initialize2 already called");
+
+        if (totalSupply() == 0) {
+            _adjuster = 1e27;
```

Comment:
Another valid approach here would be that in case of 0 `totalSupply` we could set the adjuster to rebasingCredits: 
```
_adjuster = rebasingCreditsPerTokenHighres();
```

The difference in behaviour occurs if we deposit a fresh wrapped contract after rebases on the underlying token have already occurred. In case we set the adjuster to `rebasingCreditsPerTokenHighres()` the WOETH -> OETH exchange rate would start at 1:1 and follow the underlying token's rebase rate.
If we keep it at `1e27` the exchange rate will follow the value accrual of OETH from inception.

Reply:
I think I prefer the 1e27 approach - bigger number is more accuracy? Plus we'll be deploying this soon after launch on new tokens, so the end number should be very similar.